<launch>

	<node name="map_visualizer" pkg="map_server" type="map_visualizer.py" />

	<node name="parking_car" pkg="obstacles" type="parking_car.py" />
	<node name="collision_check" pkg="obstacles" type="collision_check_with_visualizing_car.py" />
	<node name="state" pkg="update" type="state.py"/>
	
	<node name="spawn_agents" pkg="cv_agents" type="spawn_agent.py" output="screen"/>
	
	<node type="rviz" name="rviz" pkg="rviz" args="-d /home/nsclmds/catkin_ws/src/frenet_frame-and-stanley-in-rviz/src/cv_agents/rviz/kcity.rviz" />
	
</launch>
